tests/test_components.py

```diff
@@ -21,3 +32,45 @@ def test_metadata_default_values():
     metadata = Metadata()
     assert metadata.dateDownloaded is None
     assert metadata.probability is None
+
+
+def test_anchor_optional_fields():
+    Link(text="foo")
+    Link(url="https://example.com")
+
+
+def test_business_place_metadata_default_values():
+    metadata = BusinessPlaceMetadata()
+    assert metadata.dateDownloaded is None
+    assert metadata.probability is None
+    assert metadata.searchText is None
```

Comment:
@kmike I did not expect the metadata field to be different in different schemas. How do you want us to handle this on the code side? Is this approach of a item-type-specific metadata class OK? Should we try to find a generic name for this metadata variant that includes search terms?